circuit_knitting/cutting/__init__.py

```diff
@@ -86,7 +87,11 @@
     decompose_gates,
     PartitionedCuttingProblem,
 )
-from .cutting_evaluation import execute_experiments, CuttingExperimentResults
+from .cutting_evaluation import (
+    execute_experiments,
+    CuttingExperimentResults,
+)
```

Comment:
totally up to you, but this could all be put on one line again (and then this line would vanish from the diff)